--- origsrc/link-grammar-4.5.3/Makefile.am	2009-04-13 12:34:34.000000000 -0500
+++ src/link-grammar-4.5.3/Makefile.am	2009-04-27 18:56:39.842125000 -0500
@@ -3,7 +3,7 @@ SUBDIRS=link-grammar data man link-gramm
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = link-grammar.pc
 
-ACLOCAL_AMFLAGS = -I m4
+ACLOCAL_AMFLAGS = -I ac-helpers
 
 # Build and install the jar file in $(PREFIX)/share/java ...
 # XXX FIXME configure should look to see if it can find ant
--- origsrc/link-grammar-4.5.3/configure.in	2009-04-14 21:36:31.000000000 -0500
+++ src/link-grammar-4.5.3/configure.in	2009-04-27 18:56:06.420250000 -0500
@@ -27,7 +27,7 @@ AC_SUBST(LINK_MICRO_VERSION)
 AM_INIT_AUTOMAKE($PACKAGE, $VERSION)
 AM_MAINTAINER_MODE
 
-AC_CONFIG_MACRO_DIR([m4])
+AC_CONFIG_MACRO_DIR([ac-helpers])
 
 dnl Checks for programs.
 AC_ISC_POSIX
