--- origsrc/link-grammar-4.6.7/link-grammar/utilities.h	2010-04-07 12:59:45.000000000 -0500
+++ src/link-grammar-4.6.7/link-grammar/utilities.h	2010-05-16 22:53:14.331936700 -0500
@@ -13,10 +13,6 @@
 #ifndef _LINK_GRAMMAR_UTILITIES_H_
 #define _LINK_GRAMMAR_UTILITIES_H_
 
-#ifdef __CYGWIN__
-#define _WIN32 1
-#endif /* __CYGWIN__ */
-
 #ifndef _WIN32
 #include <langinfo.h>
 #endif
@@ -25,19 +21,8 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-
-#ifndef __CYGWIN__
-/* I was told that cygwin does not have these files. */ 
 #include <wchar.h>
 #include <wctype.h>
-#endif
-
-#if defined(__CYGWIN__) && defined(__MINGW32__)
-/* Some users have CygWin and MinGW installed! 
- * In this case, use the MinGW versions of UTF-8 support. */
-#include <wchar.h>
-#include <wctype.h>
-#endif
 
 #include "error.h"
 
