--- origsrc/link-grammar-4.6.4/link-grammar/utilities.h	2009-06-25 20:32:50.000000000 -0500
+++ src/link-grammar-4.6.4/link-grammar/utilities.h	2009-10-30 15:46:29.492133500 -0500
@@ -13,10 +13,6 @@
 #ifndef _LINK_GRAMMAR_UTILITIES_H_
 #define _LINK_GRAMMAR_UTILITIES_H_
 
-#ifdef __CYGWIN__
-#define _WIN32 1
-#endif /* __CYGWIN__ */
-
 #ifndef _WIN32
 #include <langinfo.h>
 #endif
@@ -26,10 +22,8 @@
 #include <stdlib.h>
 #include <string.h>
 
-#ifndef __CYGWIN__
 #include <wchar.h>
 #include <wctype.h>
-#endif
 
 #include "error.h"
 
@@ -66,24 +60,6 @@ char * strtok_r (char *s, const char *de
 #define rand_r(seedp) rand()
 #endif /* _MSC_VER || __MINGW32__ */
 
-/* CYGWIN on Windows doesn't have UTF8 support, or wide chars ... 
- * However, MS Visual C appearently does ... 
- */
-#ifdef __CYGWIN__
-#define mbrtowc(w,s,n,x)  ({*((char *)(w)) = *(s); 1;})
-#define wcrtomb(s,w,x)    ({*((char *)(s)) = ((char)(w)); 1;})
-#define iswupper  isupper
-#define iswalpha  isalpha
-#define iswdigit  isdigit
-#define iswspace  isspace
-#define wchar_t   char
-#define wint_t    int
-#define fgetwc    fgetc
-#define WEOF      EOF
-#define towlower  tolower
-#define towupper  toupper
-#endif /* __CYGWIN__ */
-
 #endif /* _WIN32 */
 
 #if defined(__sun__)
