--- origsrc/link-grammar-4.7.14/link-parser/Makefile.am	2013-05-01 20:46:07.000000000 -0500
+++ src/link-grammar-4.7.14/link-parser/Makefile.am	2013-10-01 23:03:45.472097500 -0500
@@ -10,7 +10,7 @@ if WITH_FAT_LINKS
 DEFS += -DUSE_FAT_LINKAGES
 endif
 
-INCLUDES = -I.. -I$(top_srcdir)/link-grammar $(ANSI_CFLAGS) $(LINK_CFLAGS) $(HUNSPELL_CFLAGS)
+INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/link-grammar $(ANSI_CFLAGS) $(LINK_CFLAGS) $(HUNSPELL_CFLAGS)
 
 # -----------------------------------------------------------
 # Directives to build the link-parser command-line application
