--- origsrc/link-grammar-4.8.6/link-grammar/link-grammar.def	2013-12-20 20:31:33.000000000 -0600
+++ src/link-grammar-4.8.6/link-grammar/link-grammar.def	2015-01-04 18:16:15.620801400 -0600
@@ -43,8 +43,6 @@ parse_options_set_max_parse_time
 parse_options_get_max_parse_time
 parse_options_set_cost_model_type
 parse_options_get_cost_model_type
-parse_options_set_use_fat_links
-parse_options_get_use_fat_links
 parse_options_set_use_sat_parser
 parse_options_get_use_sat_parser
 parse_options_set_use_viterbi
@@ -161,24 +159,16 @@ linkage_constituent_node_get_child
 linkage_constituent_node_get_next
 linkage_constituent_node_get_start
 linkage_constituent_node_get_end
-lg_mbrtowc
 prt_error
 parse_options_print_total_time
 extract_thin_linkage
-xfree
-count_set_effective_distance
 intersect_strings
 xalloc
 post_process_match
 match_in_connector_set
-build_deletable
 exalloc
-analyze_fat_linkage
 init_connector
-set_has_fat_down
-extract_fat_linkage
 analyze_thin_linkage
-build_effective_dist
 prune_match
 lg_strlcpy
 lg_compute_disjunct_strings
