--- origsrc/link-grammar-4.2.5/Makefile.am	2006-04-12 11:18:58.000000000 -0500
+++ src/link-grammar-4.2.5/Makefile.am	2007-12-02 16:41:24.125000000 -0600
@@ -1,3 +1,5 @@
+ACLOCAL_AMFLAGS = -I ac-helpers
+
 SUBDIRS=link-grammar data link-grammar.xcode
 
 pkgconfigdir = $(libdir)/pkgconfig
--- origsrc/link-grammar-4.2.5/link-grammar/Makefile.am	2006-04-09 08:11:19.000000000 -0500
+++ src/link-grammar-4.2.5/link-grammar/Makefile.am	2007-12-02 16:41:24.125000000 -0600
@@ -97,7 +97,7 @@
 
 bin_PROGRAMS=grammar-parse
 grammar_parse_SOURCES=parse.c
-ldadd = $(top_builddir)/link-grammar/liblink-grammar.la
+ldadd = liblink-grammar.la
 
 grammar_parse_LDADD=$(ldadd)
 
